NEWS.md

```diff
@@ -1,3 +1,41 @@
+### v8.11.0 (2022-05-11)
+
+* Log forwarding from APM agent is now possible.
+
+    * Added application log aggregator to collect logs with adaptive sampling and appropriate max samples stored.
+
+    * Added `application_logging` configuration and env vars with appropriate defaults.
+
+    * Changed default value of `application_logging.enabled` to be true.
+
+    * Set `application_logging.forwarding.enabled` to false when HSM is set.
+
+    * Enabled log forwarding by default in the example config.
+
+    * Added sent, seen and dropped metrics that collected on every harvest cycle around log lines.
+
+    * Added supportability metrics for some popular logging frameworks.
+    
+    * Added supportability metrics to record if the logging features are enabled.
+
+    * Added a storage mechanisms to transactions to keep logs until transaction ends.
+
+    * Updated log aggregator to check transaction context before adding.
+
+    * Ensured both `application_logging.enabled` and `application_logging.forwarding.enabled` are true before registering the log aggregator.
+
+* Removed distributed tracing setting from example config
+
+* Fixed a typo in lib/instrumentation/core/child_process.js
```

Comment:
this was a community PR we typically give shout outs to the PR author, you can see how we do that if you look down in the NEWS.md

Reply:
https://github.com/newrelic/node-newrelic/pull/1171
Thanks @eitoball